main.go

```diff
@@ -65,13 +68,20 @@ func main() {
 		},
 	}
 
+	log := logrus.WithFields(logrus.Fields{})
+
+	// Create a new client to communicate with CAST AI API.
+	deltaHTTPClient, err := castai.NewDefaultClient(cfg.API.URL, cfg.API.Key, logger.Level, binVersion, defaultPollTimeout)
```

Comment:
I suppose this is copy-paste typo

Reply:
Where? Can you explain, please

Reply:
`deltaHTTPClient` we don't have deltas in the controller